ToggleGen/GeneratedCode/ToggleAccessor.swift

```diff
@@ -39,24 +41,31 @@ extension ToggleAccessor {
         set { manager.set(.secure(newValue), for: ToggleVariables.encryptedToggle) }
     }
 
-    public var userDefinedBooleanToggle: Bool {
+    public var booleanToggle2: Bool {
         get { manager.value(for: ToggleVariables.booleanToggle2).boolValue! }
         set { manager.set(.bool(newValue), for: ToggleVariables.booleanToggle2) }
     }
 
-    public var userDefinedIntegerToggle: Int {
+    public var integerToggle2: Int {
         get { manager.value(for: ToggleVariables.integerToggle2).intValue! }
         set { manager.set(.int(newValue), for: ToggleVariables.integerToggle2) }
     }
 
-    public var userDefinedNumericToggle: Double {
+    public var numericToggle2: Double {
         get { manager.value(for: ToggleVariables.numericToggle2).numberValue! }
         set { manager.set(.number(newValue), for: ToggleVariables.numericToggle2) }
     }
 
-    public var userDefinedStringToggle: String {
+    public var stringToggle2: String {
```

Comment:
Script automatically changed property names. Not sure why that happened, I guess it was outdated file? @albertodebortoli

Reply:
I noticed that too when running it locally yesterday. Yes, I think it was outdated.